internal/tui/model_list.go

```diff
@@ -226,21 +320,48 @@ func (m ListModel) View() string {
 	}
 
 	m.list.Title = "How about one of these:"
-	return appStyle.Render(m.list.View())
+
+	if m.input.Focused() {
+		s += titleStyle.Render("Edit suggestion:") + "\n"
+		s += inputStyle.Render(m.input.View()) + "\n"
+		s += helpStyle.Render(m.help.View(m.keys))
+		return appStyle.Render(s)
+	}
+	s += appStyle.Render(m.list.View())
+
```

Comment:
```suggestion

	if m.input.Focused() {
		s += titleStyle.Render("Edit suggestion:") + "\n"
		s += inputStyle.Render(m.input.View()) + "\n"
		s += helpStyle.Render(m.help.View(m.keys))
		return appStyle.Render(s)
	}
	
	m.list.Title = "How about one of these:"
	s += appStyle.Render(m.list.View())
```

Nit: Bringing the title change down after the input render section I think is cleaner and clearer code.